packages/admin/src/pages/index.page.tsx

```diff
@@ -50,4 +44,23 @@ const index = () => {
   return <div>index</div>;
 };
 
+const handleError = (error: any) => {
+  console.log('Failed to verify token', error);
+
+  if (error.response?.data?.error?.message === 'User is not an admin') {
```

Comment:
Is there any other way we can assert that the user is not an admin? relying the error message is not an ideal approach.

Reply:
Sure, using the status now